aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/ApplicationServiceImpl.java

```diff
@@ -267,6 +272,182 @@ public int publishToImaging(String applicationName, long sleepDuration, boolean
+
+            if(!computeFunctionPointsProperties.isWait()) {
+                log.info("Exiting as wait flag is set to false.");
+                return Constants.RETURN_OK;
+            }
+
+            String jobStatus = logPollingProvider != null ? logPollingProvider.pollJobLog(jobGuid) : null;
+            if (jobStatus != null && jobStatus.equalsIgnoreCase(JobState.COMPLETED.toString())) {
+                log.info("Compute function points completed successfully.");
+            } else {
+                log.info("Compute function points failed.");
+                return Constants.RETURN_JOB_FAILED;
+            }
+        } catch (ApplicationServiceException e) {
+            return Constants.RETURN_APPLICATION_INFO_MISSING;
+        } catch (JobServiceException e) {
+            return Constants.RETURN_JOB_FAILED;
+        }
+
+        return Constants.RETURN_OK;
+    }
+
+    @Override
+    public int listFunctionPointRules(String applicationName, String ruleType) {
+        String applicationGuid;
+        try {
+            ApplicationCommonDetailsDto applicationCommonDetailsDto = getApplicationDetailsFromName(applicationName);
+            if (applicationCommonDetailsDto == null) {
+                log.info("Application not found.");
+                return Constants.RETURN_APPLICATION_NOT_FOUND;
+            }
+            applicationGuid = applicationCommonDetailsDto.getGuid();
+            ApplicationDto applicationDto = getApplicationDetails(applicationGuid);
+
+            if (!applicationDto.isManaged()) {
+                log.info("List rules is not available for this application. This maybe because the application has not been analyzed or the analysis is in process.");
+                return Constants.RETURN_APPLICATION_INFO_MISSING;
+            }
+
+            List<FunctionPointRuleDto> functionPointRules = restApiService.getForEntity(ApiEndpointHelper.getApplicationFunctionPointRules(applicationGuid), new TypeReference<List<FunctionPointRuleDto>>() {});
+            Map<String, List<FunctionPointRuleDto>> groupedByType = functionPointRules.stream().collect(Collectors.groupingBy(rule -> rule.getType().toLowerCase()));
+
+            if(ruleType != null && !groupedByType.containsKey(ruleType.toLowerCase())) {
+                log.error("Invalid rule type");
+                return Constants.RETURN_INVALID_PARAMETERS_ERROR;
+            }
+
+            if(groupedByType.size() > 0) {
+                System.out.println("=========================================\n");
+            }
+
+            groupedByType.forEach((type, list) -> {
+                if(ruleType != null && !ruleType.toLowerCase().equals(type)) return;
+
+                System.out.println("Type: " + type);
+                for (int i = 0; i < list.size(); i++) {
+                    if(i == list.size() - 1) {
+                        System.out.println(list.get(i).toString());
```

Comment:
log.info add some text before each line like this
![image](https://github.com/user-attachments/assets/4165ee55-9b9a-447e-92fd-0e8296482c5f)

I wanted the text the printed like this, so that the user can copy this output to a json file and it will show as a json object.
![image](https://github.com/user-attachments/assets/0659ed53-8d4a-48e4-8689-255e5be9a4fe)

Reply:
log.info/warning etc does that and will be part of the job log.
You  will need that for JENKINS for example